docs/requirements/apod/030-controlplane-bgp.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Documentation     Controlplane VMs need BGP peering with upstream devices.
+...
+...               Controlplane VM uses BGP for dynamic routing with upstream devices. 
+...               Usually it peers with 2 upstream devices for redundancy. 
+...               
+...               Controlplane VMs should use same next-hop IP addresses within management network for 
+...               BGP peering with 2 upstream devices independant of pod.
+...               
+...               Controlplane VMs use traceroute with ttl=1 for upstream device IP address discovery 
```

Comment:
Wouldn't that translate in basically using the gateway to peer? We are using static IP assignments in this net?